straxen/plugins/peaklet_processing.py

```diff
@@ -536,6 +536,18 @@ def compute(self, peaklets_he):
         return super().compute(peaklets_he)
 
 
+@export
+class MergedS2sClassification(PeakletClassification):
+    """Classify merged_s2s as unknown, S1, or S2."""
+    provides = 'merged_s2s_classification'
+    depends_on = ('merged_s2s',)
+    __version__ = '0.0.1'
+    child_plugin = True
```

Comment:
```suggestion
    child_plugin = True
    save_when = strax.SaveWhen.EXPLICIT
```
I guess this data_type is computed quickly, is not it? Maybe we can make computing's life a bit easier if we make it  a SaveWhen Explicit data_type. @ershockley would this help, or do you have to track the lineage of plugins which are not returning data by default, too? 
I am wondering if we should not do the same for PeakletClassification too. Only a few people will look at peaklet level data anyhow.

Reply:
@WenzDaniel, now the test fails at plot waveform when it asks for peak data. Maybe go back to savewhen always if it's not too much a fuss?

Reply:
We track the lineage of all data types, but it definitely would be nice to not have so many datasets running around. I would much prefer save_when to be target or explicit if possible